README.md

```diff
@@ -1,13 +1,21 @@
-## Studio Library
+<p align="center">
+  <a href="http://www.studiolibrary.com/">
+    <img
+      alt="logo"
+      src="./resource/icons/icon_white_on_blue.png"
+    />
+  </a>
+  <h1 align="center">Studio Library</h1>
+  <h4 align="center">Animation and pose manager for Maya</h4>
+</p>
 
-Studio Library is a free python script for managing poses and animation in Maya.
 
-* www.studiolibrary.com
+## Description :
```

Comment:
I don't think we need the "Description" header, so maybe just have "A Free tool ..." under the title without the header.